sklearn/ensemble/tests/test_forest.py

```diff
@@ -1167,7 +1167,7 @@ def test_class_weights(name):
 
     # Iris is balanced, so no effect expected for using 'balanced' weights
     clf1 = ForestClassifier(random_state=0)
-    clf1.fit(iris.data, iris.target)
+    clf1.fit(iris.data, iris.target, sample_weight=np.ones_like(iris.target))
```

Comment:
Please add an inline comment to explain why leaving `sample_weight=None` does not work for this test.